tests/test_sim_client.py

```diff
@@ -0,0 +1,340 @@
+# This file is part of ts_hvac.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import asynctest
+import logging
+
+from lsst.ts.hvac.simulator.sim_client import SimClient
+
+logging.basicConfig(format="%(asctime)s:%(levelname)s:%(name)s:%(message)s", level=logging.DEBUG)
+
+
+class SimClientTestCase(asynctest.TestCase):
+    async def setUp(self):
+        self.log = logging.getLogger("SimClientTestCase")
+        self.mqtt_client = SimClient()
+
+    async def test_chiller01P01(self):
+        expected_setpoint_activo = 90
+        expected_comando_encendido = True
+        await self.mqtt_client.chiller01P01(
+            setpoint_activo=expected_setpoint_activo, comando_encendido=expected_comando_encendido
+        )
+        self.assertEqual(expected_setpoint_activo, self.mqtt_client.chiller01_p01.setpointActivo)
+        self.assertEqual(expected_comando_encendido, self.mqtt_client.chiller01_p01.comandoEncendido)
+
+        expected_setpoint_activo = 110
+        try:
```

Comment:
I strongly suggest the following, instead, for clarity:
```
with self.assertRaises(ValueError):
    await self.mqtt_client.chiller01P01 (...)
```
and likewise in your other tests below.

Reply:
Nice, I didn't know that was possible. Done.